src/main/java/it/gov/pagopa/payhub/auth/mypay/service/MyPayOperatorsService.java

```diff
@@ -0,0 +1,48 @@
+package it.gov.pagopa.payhub.auth.mypay.service;
+
+import it.gov.pagopa.payhub.auth.mypay.model.MyPayOperator;
+import it.gov.pagopa.payhub.auth.mypay.repository.MyPayOperatorsRepository;
+import it.gov.pagopa.payhub.auth.utils.Constants;
+import java.util.Optional;
+import java.util.Set;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class MyPayOperatorsService {
+
+  private final MyPayOperatorsRepository myPayOperatorsRepository;
+
+  public MyPayOperatorsService(MyPayOperatorsRepository myPayOperatorsRepository){
+    this.myPayOperatorsRepository = myPayOperatorsRepository;
+  }
+
+  public void registerMyPayOperator(String mappedExternalUserId,String email, String organizationIpaCode,
+      Set<String> roles) {
+    //exit if doesn't exist either roles
+    if(!roles.contains(Constants.ROLE_ADMIN) && !roles.contains(Constants.ROLE_OPER)) {
```

Comment:
let's store the value of the following instruction and re-use it instead of calculate each time:
`roles.contains(Constants.ROLE_ADMIN)`